### ISSUE TITLE
preproduce result on monodepth2

### ISSUE THREAD

**Issue by huynhbaobk** [LINA-lln/ADDS-DepthNet#9]
Hi many thanks your hard work, I tried to train monodepth2 with oxford dataset (day and night) as Table 1. but the result quite bad. Could you show how to train monodepth2 with oxford dataset with your config?

**Comment by goxq** [comment]
@huynhbaobk Hi, I would like to know if you get the proposed method's result present in the paper? I run the code but the result is a bit far from the paper.

**Comment by huynhbaobk** [comment]
@goxq Hi, I could not get the result present in the paper. It really far. I wonder what they did.

**Comment by goxq** [comment]
@huynhbaobk Me too. The result of 'Ours' in Table 1 obtained by the pretrained model the author give is 
abs_rel | sq_rel | rmse | rmse_log | a1 | a2 | a3 | 
& 0.109 & 0.584 & 3.578 & 0.153 & 0.880 & 0.976 & 0.992 (test at day) and
& 0.233 & 2.345 & 6.860 & 0.270 & 0.632 & 0.908 & 0.962 (test at night), which is consistent with paper.

But the result of my own training is 
& 0.132 & 0.721 & 3.770 & 0.174 & 0.816 & 0.970 & 0.993 (test at day) and
& 0.284 & 2.454 & 7.079 & 0.308 & 0.507 & 0.871 & 0.958 (test at night).

How about yours? Thx a lot!!

**Comment by huynhbaobk** [comment]
This is my result 
-> Evaluating val_night 
abs_rel |   sq_rel |     rmse | rmse_log |       a1 |       a2 |       a3 | 
0.391  &   5.125  &  13.662  &   0.427  &   0.366  &   0.664  &   0.872  \\================================================== 
-> Evaluating val_day 
abs_rel |   sq_rel |     rmse | rmse_log |       a1 |       a2 |       a3 | 
0.140  &   1.224  &   6.547  &   0.205  &   0.785  &   0.952  &   0.985  \\

**Comment by huynhbaobk** [comment]
Let me upload my code. You could compare to your code

**Comment by huynhbaobk** [comment]
Hi @goxq, this is my code
https://github.com/huynhbaobk/monodepth2-oxford-robotcar

**Comment by goxq** [comment]
@huynhbaobk Seems that you don't use the code in this repo?

**Comment by huynhbaobk** [comment]
Yes, the code base on monodepth2, but I still use mainly code from ADDS
